test/tbb/test_overwrite_node.cpp

```diff
@@ -423,3 +423,39 @@ TEST_CASE("test overwrite_node try_put_and_wait") {
     test_overwrite_node_try_put_and_wait();
 }
 #endif
+
+//! Test for nested make_edge from predecessor of continue_node that adds edge to overwrite_node
+//! \brief \ref error_guessing
+TEST_CASE("Nested make_edge to continue_node") {
+    tbb::flow::graph g;
+
+    using msg_t = tbb::flow::continue_msg;
+    using cnode_t = tbb::flow::continue_node<msg_t>;
+    using wnode_t = tbb::flow::write_once_node<msg_t>;
```

Comment:
Should we test both `overwrite` and `write_once_node` here?

Reply:
Yes. If we agree the test is good then I can do the same for `overwrite_node`. `write_once_node` inherits from `overwrite_node` so its the same function, but even so, we should do both.